source/classes.tex

```diff
@@ -6335,7 +6336,9 @@
 destruction,
 the function called is the
 final overrider in the constructor's or destructor's class and not one
-overriding it in a more-derived class.
+overriding it in a more-derived class;
+if the final overrider is a pure virtual function,
```

Comment:
make that a new sentence and change: the final overrider -> that final overrider
(We're not talking about the final overrider in the entire hierarchy, but the one of the ctor's class.)

Reply:
Also add a cross-reference to the definition of "pure virtual function".